web-app-dsc-vm/azuredeploy.json

```diff
@@ -0,0 +1,423 @@
+            }
+        },
+        "modulesUrl": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the URL for downloading the PowerShell DSC module. In this case it should be the link to a zip file hosted under an Azure storage container" 
+            }
+        },
+        "sasToken": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the Shared Access Signature token used for accessing the Azure storage URL linked to the PowerShell DSC module" 
+            }
+        },
+        "webdeploypkg": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the link to the Web Deploy package for the website that's going to be deployed to the Virtual Machine" 
+            }
+        },
+        "configurationFunction": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is the link to the Web Deploy package to be deployed to the Virtual Machine. It is a github URL in this example." 
+            }
+        },
+        "serverName": {
+            "type": "string",
+             "metadata": { 
+                "Description": "This is database server name" 
+            }
+        },
+        "serverLocation": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is database server location" 
+            }
+        },
+        "administratorLogin": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is database server administrator login name" 
+            }
+        },
+        "administratorLoginPassword": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is database server administrator login password" 
+            }
+        },
+        "databaseName": {
+            "type": "string",
+            "metadata": { 
+                "Description": "This is name of the database hosted in the database server" 
+            }
+        },
+        "collation": {
+            "type": "string",
+            "defaultValue": "SQL_Latin1_General_CP1_CI_AS",
+            "metadata": { 
+                "Description": "This is database collation - rule for comparing the encodings in the database" 
+            }
+        },
+        "edition": {
+            "type": "string",
+            "defaultValue": "Web",
+            "metadata": { 
+                "Description": "This is the edition of the database server" 
+            }
+        },
+        "maxSizeBytes": {
+            "type": "string",
+            "defaultValue": "1073741824",
+            "metadata": { 
+                "Description": "This is the maximum database size in bytes for Microsoft Azure SQL Database" 
+            }
+        },
+        "requestedServiceObjectiveId": {
```

Comment:
'RequestedServiceObjectiveId' is not required anymore. I have revised the template accordingly.